CMakeLists.txt

```diff
@@ -65,22 +64,6 @@ check_include_file(strings.h HAVE_STRINGS_H)
 if( HAVE_STRINGS_H AND NOT HAVE_STRING_H )
   set(NEED_BSD_STRINGS 1)
 endif()
-check_type_size(size_t SIZE_T) # -> HAVE_SIZE_T
-if( NOT HAVE_SIZE_T )
-  message(WARNING "Type size_t is not defined in any of the usual places.
-    Try putting 'typedef unsigned int size_t;' in jconfig.h."
-  )
-endif()
-check_type_size("unsigned char" UNSIGNED_CHAR) # -> HAVE_UNSIGNED_CHAR
```

Comment:
Used here: https://github.com/hunter-packages/jpeg/blob/019f51b1964216949a9b902c79fb4975cd3c98c6/jconfig-cmake.cfg#L5

Reply:
Doesn't that mean it is hardcoded to 1 and will become:
#define HAVE_UNSIGNED_CHAR 1

Reply:
> Doesn't that mean it is hardcoded to 1 and will become

No, see how `configure_file` works:
* https://cmake.org/cmake/help/latest/command/configure_file.html